Corgibytes.Freshli.Cli/Functionality/ReadCycloneDxFile.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using PackageUrl;
+
+namespace Corgibytes.Freshli.Cli.Functionality;
+
+public class ReadCycloneDxFile
+{
+    private readonly IFileReader _fileReader;
+
+    public ReadCycloneDxFile(IFileReader fileReaderService) => _fileReader = fileReaderService;
+
+    public List<PackageURL> AsPackageUrls(string filePath)
+    {
+        if (filePath == "")
+        {
+            throw new ArgumentException("Can not read file, as no file path was given");
+        }
```

Comment:
Let's turn this into a localized string and use the `_ filePath ?? ...` construction.